src/controllers/dynakube/istio/istio.go

```diff
@@ -6,86 +6,49 @@ import (
 	"fmt"
 	"net"
 
-	"k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/api/meta"
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-	"k8s.io/client-go/discovery"
-	"k8s.io/client-go/rest"
 )
 
 var (
-	istioGVRName = "networking.istio.io"
+	IstioGVRName = "networking.istio.io"
```

Comment:
does this need to be exported?

Reply:
No, except for CheckIstioEnabled no function/variable stated from you needed to be exported, changed it, thank you